wp-content/civi-extensions/goonjcustom/Civi/DroppingCenterService.php

```
@@ -738,4 +741,205 @@ private static function isViewingDroppingCenter($tabsetName, $context) {
+      ->addSelect('email.email', 'display_name')
+      ->addJoin('Email AS email', 'LEFT')
+      ->addWhere('id', '=', $initiatorId)
+      ->execute()->single();
+
+    $attendeeEmail = $campAttendedBy['email.email'];
+    $attendeeName  = $campAttendedBy['display_name'];
+
+    // Define last month’s range.
+    $startDate = (new \DateTime('first day of last month'))->format('Y-m-d');
+    $endDate   = (new \DateTime('last day of last month'))->format('Y-m-d');
+    $monthName = (new \DateTime('first day of last month'))->format('F Y');
```

PHPMD (2.15.0)

Avoid unused local variables such as '$monthName'. (undefined)

(UnusedLocalVariable)